HW0/P4/P4.py

```diff
@@ -40,38 +46,125 @@
     # Read the observation array and plot it (Part 2)
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    data2 = np.loadtxt('P4_measurements.txt', delimiter=",")
+    C = np.matrix([[1.0/rx, 0, 0], [0, 1.0/ry, 0], [0, 0, 1.0/rz]])
+    #print C
```

Comment:
you have a lot of commented out 'print ___' lines in this file.  might be worth removing them!